docker-compose.yml

```diff
@@ -1,23 +1,63 @@
-version: '3.4'
-
 networks:
   altinnplatform_network:
     external: false
 
 services:
-  altinn_platform_storage:
-    container_name: altinn-platform-storage
-    image: altinnplatformstorage:latest
-    restart: always
+  # altinn_platform_storage:
+  #   container_name: altinn-platform-storage
+  #   image: altinnplatformstorage:latest
+  #   restart: always
+  #   networks:
+  #     - altinnplatform_network
+  #   environment:
+  #     - ASPNETCORE_ENVIRONMENT=Development
+  #     - ASPNETCORE_URLS=http://+:5010
+  #     - PlatformStorageSettings:ApiEndPoint=http://altinn-platform-storage:5010/api/v1
+  #     - PlatformStorageSettings:ApiEndPointHost=http://altinn-platform-storage:5010/
+  #   ports:
+  #     - "5010:5010"
+  #   build:
+  #     context: .
+  #     dockerfile: Dockerfile
+
+  altinn_platform_storage_postgres:
+    image: postgres:16
+    container_name: storage_postgres
+    restart: unless-stopped
     networks:
       - altinnplatform_network
     environment:
-      - ASPNETCORE_ENVIRONMENT=Development
-      - ASPNETCORE_URLS=http://+:5010
-      - PlatformStorageSettings:ApiEndPoint=http://altinn-platform-storage:5010/api/v1
-      - PlatformStorageSettings:ApiEndPointHost=http://altinn-platform-storage:5010/
+      - POSTGRES_USER=platform_storage_admin
+      - POSTGRES_PASSWORD=Password
+    ports:
+      - '5432:5432'
```

Comment:
This will come in conflict for us that already have PostgreSQL running on out machines.